packages/cli-doctor/src/tools/healthchecks/androidSDK.ts

```diff
@@ -50,8 +50,8 @@ const getBuildToolsVersion = (projectRoot = ''): string => {
       .substring(buildToolsVersionIndex)
       .split('\n')[0]
       // Get only the the value of `buildToolsVersion`
-      .match(/\d|\../g) || []
-  ).join('');
+      .match(/\d+/g) || []
```

Comment:
if the version is always three parts, shouldn't we do `/\d+\.\d+\.\d+/g`? 🤔 
This probably allow to drop the `join` below as well.

Reply:
Good catch. Build tools versions are `\d+\.\d+\.\d+`. PR updated.